src/types.ts

```diff
@@ -255,6 +255,23 @@ export type BehaviorRegistry = {
   [key: string]: HvBehavior;
 };
 
+// https://hyperview.org/docs/reference_behavior_attributes
+export const ATTRIBUTES = {
```

Comment:
might make sense to rename to `BEHAVIOR_ATTRIBUTES` to remove ambiguity since this file is central for types

Reply:
Funny, I had that and was about to commit it but because "TRIGGERS" and others were not named such, I backed out. I'll add that back in.